htdocs/landing/aup.html

```diff
@@ -0,0 +1,104 @@
+<!doctype html>
+<html>
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+  <title>GOCDB Privacy Notice</title>
+  <link rel="stylesheet" type="text/css" href="home.css" />
+  <link rel="SHORTCUT ICON" href="Logo-1.4-FavIcon-32x32.regional.ico" />
+</head>
+
+<div class="wide_page_container" >
+  <div class="centre_box">
+    <div class="centrePageContainer">
+      <div class="policyText">
+        <!--Start of policy text output by PANDOC -->
```

Comment:
Is this useful to note?

Reply:
Yes. The lines between !Start and !End are (almost) auto-generated from the EGI .docx file. Instructions for this are in the Admin Guide